main.py

```diff
@@ -711,6 +717,52 @@ def make_positive_or_negative(self) -> None:
 
             self._positive_or_negative_json["data"].append(data)
 
+    def make_warning_and_phase(self) -> None:
+        # warning_and_phase.jsonを生成する
+        self._warning_and_phase_json = {
+            "data": {},
+            "last_update": self.get_inspections_last_update()
+        }
+
+        warning = 0
+        phase = 0
+
+        positive_or_negative_dict = self.positive_or_negative_json()
+        average_length = len(positive_or_negative_dict['data'])
+        latest_average_patients = positive_or_negative_dict['data'][average_length - 1]['7日間平均陽性数']
+
+        soup = get_html_soup(base_url)
+
+        real_page_tags = soup.find_all("p", align="center")
+        
+        for tag in real_page_tags:
+            strong = tag.find("strong")
+            if strong != None:
+                strong_text = strong.get_text()
+                if ("感染拡大特別期" in strong_text) and ("【" in strong_text) and ("】" in strong_text):
+                    warning = 5
+        
+        if warning == 5:
+            phase = 2
+        elif latest_average_patients >= 40:
+            warning = 4
+            phase = 2
+        elif latest_average_patients >= 30:
+            warning = 3
+            phase = 2
+        elif latest_average_patients >= 20:
+            warning = 2
+            phase = 2
+        elif latest_average_patients >= 10:
+            warning = 1
+            phase = 1
+        else:
+            warning = 0
+            phase = 0
+
+        self._warning_and_phase_json['data']['警戒基準'] = warning
+        self._warning_and_phase_json['data']['対応の方向性'] = phase
```

Comment:
```suggestion
        self._warning_and_phase_json["data"]["警戒基準"] = warning
        self._warning_and_phase_json["data"]["対応の方向性"] = phase
```
こちらもダブルクオーテーションで統一します